src/node.rs

```diff
@@ -2,15 +2,17 @@
 
 #[cfg(not(feature = "deterministic"))]
 use std::collections::{hash_map, HashMap};
-use std::collections::{hash_set, HashSet};
 use std::fmt;
 use std::ops::Deref;
+use std::slice::Iter as SliceIter;
 
 use html5ever::tendril::StrTendril;
 use html5ever::{Attribute, LocalName, QualName};
 
 use selectors::attr::CaseSensitivity;
 
+use once_cell::unsync::OnceCell;
```

Comment:
If we go for this, whether this uses `unsync::OnceCell` or `sync::OnceCell` should probably depend on the `atomic` feature proposed in #102.

Reply:
Actually not correct since we only care about `Send`, not `Sync` and hence even `unsync::OnceCell` is sufficient for `Html` to stay `Send`.